cmake/pheromones/common.cmake

```diff
@@ -0,0 +1,43 @@
+include(FetchContent)
+
+# Adding modules
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH})
+
+function(pheromones_fetch_populate)
+    if(PHEROMONES_PATH)
+        message(INFO "PHEROMONES_PATH specified, skipping fetch")
+    endif()
+
+    FetchContent_Declare(
+        pheromones
+        GIT_REPOSITORY https://github.com/SwarmUS/Pheromones
+        GIT_TAG        01bd792711588c425fe0d5e540cbdcff746f5fec
```

Comment:
Maybe change to branch master instead of tag? To make sure to always have the newest version on the build.

Reply:
I would maybe let it a tag, if not, updating pheromones could break your build. Ex during a Hivemind PR your build could break because pheromones was updated